packages/x-data-grid/src/components/virtualization/GridVirtualScrollbar.tsx

```diff
@@ -139,12 +148,12 @@ const GridVirtualScrollbar = forwardRef<HTMLDivElement, GridVirtualScrollbarProp
     useOnMount(() => {
       const scroller = apiRef.current.virtualScrollerRef.current!;
       const scrollbar = scrollbarRef.current!;
-      const options = { capture: true, passive: true };
+      const options = { passive: true };
```

Comment:
```suggestion
      const options: AddEventListenerOptions = { passive: true };
```

Reply:
`capture: true` was suspicious, happy we could remove this.